profiles/schema.py

```diff
@@ -370,6 +371,46 @@ def filter_by_nin_exact(self, queryset, name, value):
             return queryset.none()
 
 
+class AllowedDataFieldsMixin:
```

Comment:
Not sure if the correct place to subclass this would be on the graphene node classes or on the Django `Model` classes. :thinking:

Reply:
Justifying my decision; my thought was to lean more for the api side since I think model should know as little as possible for the api and its visibilities => handle the visibility checks/permissions in the api level not in model => schema/nodes

Reply:
changed to model solution. It truly is more efficient that way 👌